### ISSUE TITLE
3D Headless mode running same plane several times?

### ISSUE THREAD

**Issue by andrewsilberfeld** [MouseLand/cellpose#806]
Hello,

I'm running a test image using cellpose in headless version and for some reason, it runs the image 4 times in the YX, 4 times in the XZ, and 4 times in the YZ planes. Is this a bug or is there something in my command that I'm doing causing it to do that? I should also add that this is the greatest tool I've ever seen. I'm so impressed with the segmentation accuracy in a really challenging embryonic setting!

I ran it using this command
cellpose --image_path /n/path/to/my/images/cellpose/tmp_small.tif --pretrained_model nuclei --do_3D --cellprob_threshold -5 --diameter 10 --save_tif --save_rois --verbose

![image](https://github.com/MouseLand/cellpose/assets/62805456/dea5b2bc-b7fb-4b66-ae5b-1e78d513836c)

**Comment by carsen-stringer** [comment]
this is the neural network and then the dynamics running
